src/components/RichTextEditor/Plugins/MentionPlugin/MentionCombobox/MentionComboboxItem.tsx

```diff
@@ -0,0 +1,47 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React from 'react';
+import { ComboboxItemProps } from '@udecode/plate';
+import { MentionItemData, MentionableCategory } from '../types';
+import { IconPeople, IconPerson, IconSize, IconTarget } from '@foundation/Icon';
+
+type RenderAvatarProps = {
+    image: string | undefined;
+    category: MentionableCategory;
+    text: string;
+    id: string;
+};
+
+const RenderImage = ({ image, category, text, id }: RenderAvatarProps) => {
+    let avatar;
+    switch (category) {
+        case MentionableCategory.GROUP:
+            avatar = <IconPeople size={IconSize.Size12} />;
+            break;
+        case MentionableCategory.ALL:
+            avatar = <IconTarget size={IconSize.Size12} />;
+            break;
+        default:
+            avatar = <IconPerson size={IconSize.Size12} />;
+            break;
+    }
+
+    return image ? <img src={image} alt={`${id}-${category}-${text}`} /> : avatar;
+};
+
+export const MentionComboboxItem = ({ item }: ComboboxItemProps<MentionItemData>): JSX.Element => {
```

Comment:
Prefer `ReactElement` over `JSX.Element`
```suggestion
export const MentionComboboxItem = ({ item }: ComboboxItemProps<MentionItemData>): ReactElement => {
```

Reply:
Updated